lib/releases.sh

```diff
@@ -35,7 +35,8 @@ export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v0
 # CAPM3 releases
 if [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
   export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v0.4.")}"
-# TODO: add else condition when v1alpha5 is available.
+else
+  export CAPM3RELEASE="${CAPM3RELEASE:-"v0.5.0")}"
```

Comment:
This is not valid` v0.5.0`

Reply:
That was a test since we are using local overrides, we didnt want to use the release version that matches with v1a4 release but it seems we missed something else.